docs/scenarios/library-authoring/pinning-libraries-to-rc2.md

```diff
@@ -0,0 +1,70 @@
+# How to manage package dependency versions for RC2
+
+By [Phillip Carter](https://github.com/cartermp)
+
+This article covers how to manage package versions when authoring a library for .NET Core.
+
+## Glossary
+
+**TFM** - Target Framework Moniker.  This is a string that represents the target you want your code to run on.  In this case of a library, this is `netstandard1.x`, where `1 <= x <= 5`.
+
+**Metapackage** - A NuGet package that represents a set of NuGet packages.  They're organized in a way that makes sense for general library and app development.
+
+## Pin your code to RC2
```

Comment:
Good call.  "Pinning" does have some negative connotations.  It also may not be a clear word for non-English speakers.  "Snap" is a good alternative, but I think I may go with "fasten" because it feels like it's a bit less ambiguous.